src/govuk/components/input/input.yaml

```
@@ -2,15 +2,15 @@ params:
 - name: id
   type: string
   required: true
-  description: The id of the input.
+  description: The ID of the input.
 - name: name
   type: string
   required: true
   description: The name of the input, which is submitted with the form data.
 - name: type
   type: string
   required: false
-  description: Type of input control to render. Defaults to "text".
+  description: Type of input control to render. Defaults to 'text'.
```

Again, wondering if this should possibly be marked up as code as it has 'meaning' in HTML…

```suggestion
  description: Type of input control to render. Defaults to `text`.
```